Makefile

```diff
@@ -621,12 +622,24 @@ test-rules: \
 #
 	$(CMD_OPA) test $(REGO_SIGNATURES_DIR) --verbose
 
+
+#
+# types
```

Comment:
Despite this change being functional and working good, I have concerns on "test-types". What does it mean for the end user ? What is "types" ?  We had, so far, a clear distinction between "tracee-ebpf", "tracee-rules" and "rules", thus the 3 different tests. 

Shouldn't we add the types test, for now, to "test-tracee-ebpf" OR "test-tracee-rules" ? And, after we're done with making "tracee-ebpf + tracee-rules" a single program, then we can have all tests in a single "test target".

WDYT ?

Reply:
I'm not sure it fits either programs, `types` defines their protocol. We could split the inner packages between testing ebpf and rules but that seems ugly.

I agree we can find a better solution once we merge the two into a single program, but rn I can't really thing of a "prettier" solution, because there are certainly tests here that need to run.

Reply:
I don't want to block the change, just change it to make more sense. What if we change test-types for something more generic. Check "test-tracee-ebpf"... it contains:

```
./cmd/tracee-ebpf/...
./pkg/...
```

@danielpacak had suggested we included ./pkg/ tests in tracee-ebpf for now and later organized the pkg tests into a single "test" target. @danielpacak WDYT here ?

Reply:
BTW, isn't this already being tested by that target ? with "./pkg/..." ?

Reply:
> BTW, isn't this already being tested by that target ? with "./pkg/..." ?

No since this is in a different module.
I guess i'd slightly lean on this being in tracee-rules tests, but since `trace` is relevant to tracee-ebpf that doesn't make too much sense either. I can't really see this not being tested seperately for now.

Maybe we need to split tests to ebpf cmd rules cmd and pkg and we can add this under pkg? But that's a larger scope than this PR.

Reply:
yes, let's do the following: to add the test into an existing target (tracee-ebpf or tracee-rules) and open an issue to either:

1. create a single test target containing all needed tests
2. or to create multiple targets to each pkg (with a broader one executing them all)

WDYT ?

Reply:
(then you can get rid of the workflow changes as well)

Reply:
1. That exists in `make test` which just composes all the other test targets.

I think we just missed restructuring the tests in terms of the new latest structure. Again that would be outside this scope. In terms of the current makefile structure I think how it's done here makes the most sense (at least short term). A PR fixing the entire structure could use the changes done here and reimplement them but I fear if we don't merge this now it will be missed along the way later on.

If we have to include this in one of the other test targets... I guess it would be tracee rules, but I think it should be kept in it's own target.

Reply:
Okay, we've synced on Teams about this. We agreed you would include the test in `test-tracee-rules` target and I'm going to open an issue to reformulate the tests targets with the new project structure (since `test-tracee-ebpf` and `test-tracee-rules` are not reflecting the project structure we have today).

Reply:
https://github.com/aquasecurity/tracee/issues/1522